modules/scylladb/scylladb.go

```diff
@@ -0,0 +1,214 @@
+package scylladb
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/docker/go-connections/nat"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	port           = "9042/tcp"
+	shardAwarePort = "19042/tcp"
```

Comment:
@danielhe4rt in the Java implementation, the alternator port is a constant set to `8000`. In Go, we allow customising this port. We'd like to align both implementations, so I'd like you to help us understand the rationale behind making it fixed Vs dynamic. 🙏

Reply:
@eddumelendez I noticed the SSL implementation in Java is hardcoding the paths for the certificates and trustStore: https://github.com/testcontainers/testcontainers-java/blob/5af66d700e942474afabd8e550bf089196fdb5f9/modules/scylladb/src/main/java/org/testcontainers/scylladb/ScyllaDBContainer.java#L77-L83 But those values are set in a test file, the scylladb config: https://github.com/testcontainers/testcontainers-java/blob/5af66d700e942474afabd8e550bf089196fdb5f9/modules/scylladb/src/test/resources/scylla-test-ssl/scylla.yaml#L463-L468

If users have different paths set in their test files, then they cannot update them in the code. Is this a potential bug?

Reply:
@danielhe4rt @eddumelendez I'm taking the decision of pinning the alternator port to the 8000 default port, not allowing its customisation. This is how the Java implementation is done, so I'm aligning with that.

PLMK if you have any concern about it, specially if you prefer it to be configurable (as it was in this current state)

Reply:
About the port, there is no specific reason. IIRC I used 8000 because is the one used in scylladb docs. Regarding to the certificates and truststore, I think there is one issue with it now because we are forcing format instead to be open.